cmd/cmdapi/cmdapi.go

```diff
@@ -45,7 +45,7 @@ func main() {
 
 	versionHandler := func() http.HandlerFunc {
 		return func(w http.ResponseWriter, r *http.Request) {
-			w.Write([]byte(version))
+			w.Write([]byte(`{"version":"` + version + `"}`))
```

Comment:
add a JSON header, optionally over-optimize by doing the byte concat just once (rather than every call). like e.g. https://github.com/micromdm/nanomdm/blob/main/http/http.go#L43-L50